### DISCUSSION TITLE
How to add events on keys

### DISCUSSION THREAD

**Discussion by Hasheditz** [COST-SD/support#16] [Q&A] [answered]
Since I am creating a js site where I have to add some sort of keybinding can anyone help to set that?

**Comment by getlost01** [comment, marked as answer]
Here is an example of how you can set up keybinding using the `keyup` event in JavaScript:

You can also use `event.key` to check what key has been pressed.
```
document.addEventListener('keyup', function(event) {
    if (event.key === 'a') {
        // Do something when the 'a' key is pressed
    } else if (event.key === 'b') {
        // Do something when the 'b' key is pressed
    }
});
```
This code will also listen for the `keyup` event on the entire document, and check the event.key property to see which key was pressed.
